lib/ui/components/Transitions/Fade.js

```diff
@@ -19,7 +18,7 @@ function Fade({
     exit: getCSSVariableAsNumber('--animation-duration-fast-ms'),
   };
 
-  const defaultEasing = getCSSVariableAsObject('--animation-easing-js');
+  const defaultEasing = '--animation-easing-js';
```

Comment:
This won't work here, I don't think? Did you test it? I'm not sure where you got the idea that animejs accepts a css variable for the easing parameter, but [it doesn't seem like it does](https://animejs.com/documentation/#staggeringEasing).

What I suggest we do is to replace this with `cubicBezier(0.4, 0, 0.2, 1)` (which should correspond to the CSS easing value). Yes it is duplication. No, it isnt DRY. Its fine here.

Ultimately, we want to delete this file and remove animejs as a dependency, too.